### DISCUSSION TITLE
Add Budgets and Labels

### DISCUSSION THREAD

**Discussion by Roxas2712** [victorbalssa/abacus#23] [Ideas]
Hello,

as I already mentioned I would like to use abacus to log all my purchases etc. remotely :)

therefore it would be great to also label my transactions or add them to a specificity budget directly within the app. 

Do you think this would be possible?

**Comment by warllo54** [comment]
I would really appreciate the budget/ category options as well.

**Reply by danielnbalasoiu** [reply]
As a workaround I added automation rules for different/**some** expense accounts to assign budgets and categories wheneven I create a transaction **but it would be awesome if this feature is implemented.**

**Comment by victorbalssa** [comment]
Thanks for your feedback!
It really helps me prioritize the next update.
#24 will be present in the next release.
